wait-for-it.sh

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
+#   Use this script to test if a given TCP host/port are available
+
```

Comment:
I guess this one should be deleted, and we keep `docker_prestashop/wait-for-it.sh` instead?